diff --git a/chrome/app/theme/chrome_unscaled_resources.grd b/chrome/app/theme/chrome_unscaled_resources.grd
index c8fb1b8e88..2cffbfc0e3 100644
--- a/chrome/app/theme/chrome_unscaled_resources.grd
+++ b/chrome/app/theme/chrome_unscaled_resources.grd
@@ -88,10 +88,10 @@
               <include name="IDR_PRODUCT_LOGO_ANIMATION_SVG" file="${branding_path_component}/product_logo_animation.svg" type="BINDATA" />
             </else>
           </if> <!-- _is_chrome_for_testing_branded -->
-            <include name="IDR_PRODUCT_LOGO_16_SHORTCUTS" file="chromium/product_logo_16.png" type="BINDATA" />
-            <include name="IDR_PRODUCT_LOGO_24_SHORTCUTS" file="chromium/product_logo_24.png" type="BINDATA" />
-            <include name="IDR_PRODUCT_LOGO_64_SHORTCUTS" file="chromium/product_logo_64.png" type="BINDATA" />
-            <include name="IDR_PRODUCT_LOGO_128_SHORTCUTS" file="chromium/product_logo_128.png" type="BINDATA" />
+            <include name="IDR_PRODUCT_LOGO_16_SHORTCUTS" file="${branding_path_component}/product_logo_16.png" type="BINDATA" />
+            <include name="IDR_PRODUCT_LOGO_24_SHORTCUTS" file="${branding_path_component}/product_logo_24.png" type="BINDATA" />
+            <include name="IDR_PRODUCT_LOGO_64_SHORTCUTS" file="${branding_path_component}/product_logo_64.png" type="BINDATA" />
+            <include name="IDR_PRODUCT_LOGO_128_SHORTCUTS" file="${branding_path_component}/product_logo_128.png" type="BINDATA" />
         </else>
       </if> <!-- _google_chrome -->
       <!-- NOTE: product_logo_*.* files beyond what's listed above are
